independent-publisher-connectors/Safaricom Mpesa Daraja/apiDefinition.swagger.json

```diff
@@ -0,0 +1,620 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "daraja",
+    "description": "A connector that exposes daraja mpesa endpoints",
+    "version": "1.0",
+    "contact": 
+    {
+      "name": "John Muchiri, Bernard Karaba",
+      "url": "https://johnmuchirim.github.io/",
+      "email": "jeanfrancaiseharicot@gmail.com",
+      "email": "bkaraba14@gmail.com"
+    }
+  },
+  "host": "sandbox.safaricom.co.ke",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+   "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/oauth/v1/generate": {},
```

Comment:
@johnmuchiriM can you please explain the empty operations? is it possible to remove it?
Please fix all empty paths in the swagger file.